File: 백준/품/bronze/2562_bronze3_최댓값.py
# arr=[]

# for i in range(1,10):
#     arr.append(int(input()))

# print(max(arr), arr.index(max(arr))+1, sep="\n")


arr = list(int(input()) for i in range(1,10))

print(max(arr), arr.index(max(arr))+1, sep="\n")

File: 백준/품/bronze/2562_bronze3_최댓값.js
const fs = require("fs");
const input = fs.readFileSync("test.txt").toString().split("\n");

let biggest;
let index;
for (let num of input) {
  const number = Number(num);
  if (!biggest) {
    biggest = number;
    index = input.indexOf(num) + 1;
  } else if (number > biggest) {
    biggest = number;
    index = input.indexOf(num) + 1;
  }
}

console.log(biggest);
console.log(index);
